src/Modules/Data/Commands.Data/GetPowerBITable.cs

```diff
@@ -0,0 +1,116 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+using Microsoft.PowerBI.Common.Abstractions;
+using Microsoft.PowerBI.Common.Abstractions.Interfaces;
+using Microsoft.PowerBI.Common.Api.Datasets;
+using Microsoft.PowerBI.Common.Api.Workspaces;
+using Microsoft.PowerBI.Common.Client;
+
+namespace Microsoft.PowerBI.Commands.Data
+{
+    [Cmdlet(CmdletVerb, CmdletName, DefaultParameterSetName = DatasetIdSetName)]
+    [OutputType(typeof(IEnumerable<Table>))]
+    public class GetPowerBITable : PowerBIClientCmdlet, IUserFirstSkip
+    {
+        public const string CmdletVerb = VerbsCommon.Get;
+        public const string CmdletName = "PowerBITable";
+
+        #region ParameterSets
+        private const string DatasetIdSetName = "DatasetId";
+        private const string DatasetSetName = "Dataset";
```

Comment:
I think you based these parameter sets on one of the Workspace cmdlets. The Data cmdlets are newer and use a slightly different naming convention and structure. Consider updating your parameter sets to match the other Data cmdlets.

Reply:
Got it. I did DatasetIdParameterSetName and DatasetSetNameParameter